platformio.ini

```diff
@@ -1,49 +1,20 @@
 ; PlatformIO Project Configuration File
 ;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-;-------------------------------------------------------
-; global PIO parameters                                |
-;-------------------------------------------------------
-[platformio]
-;default build enfironment for PIO project
-default_envs = esp32-s3-devkitc-1
-
-;Path to source files relative to PIO project
-; see https://docs.platformio.org/en/stable/projectconf/sections/platformio/options/directory/src_dir.html
-src_dir = src
-
-;switch "include" path to be src as well, as we decide to have headers and sources in same directories
-; this prevents the build environment from complaining the path not existing
-include_dir = src
```

Comment:
maybe check this, if it was necessary to stop PlatformIO from complaining, about missing "include" directory.